cmd/registry/cmd/upload/artifact.go

```diff
@@ -83,23 +83,23 @@ func buildArtifact(ctx context.Context, parent string, filename string) (*rpc.Ar
 	jsonBytes, _ := yaml.YAMLToJSON(yamlBytes) // to use protojson.Unmarshal()
 	var artifact *rpc.Artifact
 	switch header.Kind {
-	case "DisplaySettings", patch.DisplaySettingsMimeType:
```

Comment:
Including the mime types in the Kind field was (IMO) a mistake - I think if we include anything other than the message name, it should be the fully-qualified message name to resolve any ambiguities (which don't exist now because currently all message names are unique)